tools/audioconv64/audioconv64.c

```diff
@@ -150,8 +150,11 @@ void walkdir(char *inpath, char *outpath, void (*func)(char *, char*)) {
 			if (!strcmp(de->d_name, ".") || !strcmp(de->d_name, ".."))
 				continue;
 			char *inpathsub, *outpathsub;
-			asprintf(&inpathsub, "%s/%s", inpath, de->d_name);
-			asprintf(&outpathsub, "%s/%s", outpath, de->d_name);
+			int ret;
+			ret = asprintf(&inpathsub, "%s/%s", inpath, de->d_name);
+			assert(ret != -1);
+			ret = asprintf(&outpathsub, "%s/%s", outpath, de->d_name);
+			assert(ret != -1);
```

Comment:
I'd rather add a `(void)` in front. I prefer shorter code that will segfault under extreme memory pressure, rather than noisy code that will still crash under memory pressure but with an assertion message (assuming you terminal is able to refresh...).